### PR TITLE
circular  Avtar and square Avtar

### PR DESCRIPTION
**Pull request by PRABgit01**
### Square avatar uses basic props to get value 
Screenshot 2023-12-20 at 5 42 13 PM

### circular avatar 

Screenshot 2023-12-20 at 5 42 01 PM

### CODE REVIEW

File: src/stories/Avatar/Avatar.stories.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  title: "Avatr ",
+  component: Avatar,
+};
+
+export const CircularProfileAvatar = () => (
+  <AvtarConatiner>
+    <Avatar
+      url="https://i.pravatar.cc/40"
+      type="small"
+      alt="avatar-2"
```
Comment (ankitadrixus08) [inline src/stories/Avatar/Avatar.stories.js:15]: don't give alt tag like avatar-2 please make proper name @PRABgit01

File: src/stories/Avatar/Avatar.stories.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      type="large"
+      alt="avatar-3"
+      variant="circle"
+    />
+  </AvtarConatiner>
+);
+
+export const SquareProfileAvatar = () => (
+  <AvtarConatiner>
+    <Avatar url="https://i.pravatar.cc/10" type="small" alt="avatar-1" />
```
Comment (ankitadrixus08) [inline src/stories/Avatar/Avatar.stories.js:35]: same as above @PRABgit01

File: src/stories/theme.js [hunk 3]
```diff
@@ -17,5 +17,23 @@ const theme = {
...
+},
+medium:{
+    width:'60px',
+    height: '60px'
+}
+,
+
+large:{
+    width:'60px',
+    height:'80px'
```
Comment (ankitadrixus08) [inline src/stories/theme.js:32]: use same as width @PRABgit01

File: src/stories/Avatar/Avatar.stories.js (hunk 1 above)
Comment (PRABgit01) [reply src/stories/Avatar/Avatar.stories.js:15]: ok sir i ll give proper naming to it

File: src/stories/theme.js (hunk 3 above)
Comment (PRABgit01) [reply src/stories/theme.js:32]: sir this is extra file i have done this changes in style only

**Review by ankitadrixus08 — APPROVED** [review]

File: src/stories/Avatar/Avatar.stories.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  title: "Avatr ",
+  component: Avatar,
+};
+
+export const CircularProfileAvatar = () => (
+  <AvtarConatiner>
+    <Avatar
+      url="https://i.pravatar.cc/40"
+      type="small"
+      alt="CircleAvatar1"
```
Comment (ankitadrixus08) [inline src/stories/Avatar/Avatar.stories.js:15]: CircleAvatar1 > circleAvatarSmall

**Review by ankitadrixus08 — APPROVED** [review]

File: src/stories/Avatar/Avatar.stories.js (hunk 4 above)
Comment (PRABgit01) [reply src/stories/Avatar/Avatar.stories.js]: sir did the changes

**Pull request merged**